src/components/views/WaiterOrder/WaiterOrder.js

```diff
@@ -1,23 +1,393 @@
+          basil: {
+            label: 'Fresh basil',
+            price: 2,
+            default: true,
+          },
+          salami: {
+            label: 'Salami',
+            price: 3,
+          },
+        },
+      },
+      crust: {
+        label: 'pizza crust',
+        type: 'select',
+        options: {
+          standard: {
+            label: 'standard',
+            price: 0,
+            default: true,
+          },
+          thin: {
+            label: 'thin',
+            price: 2,
+          },
+          thick: {
+            label: 'thick',
+            price: 2,
+          },
+          cheese: {
+            label: 'cheese in edges',
+            price: 5,
+          },
+          wholewheat: {
+            label: 'wholewheat',
+            price: 3,
+          },
+          gluten: {
+            label: 'with extra gluten',
+            price: 0,
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'salad',
+    name: `Nonno Alberto's Salad`,
+    price: 9,
+    params: {
+      ingredients: {
+        label: 'Ingredients',
+        type: 'checkboxes',
+        options: {
+          cucumber: {
+            label: 'Cucumber',
+            price: 1,
+            default: true,
+          },
+          tomatoes: {
+            label: 'Tomatoes',
+            price: 1,
+            default: true,
+          },
+          black_olives: {
+            label: 'Olives',
+            price: 1,
+            default: true,
+          },
+          feta: {
+            label: 'Feta cheese',
+            price: 1,
+          },
+          cheese: {
+            label: 'Cheese',
+            price: 1,
+          },
+          herbs: {
+            label: 'Fresh herbs',
+            price: 1,
+            default: true,
+          },
+          pepper: {
+            label: 'Black pepper',
+            price: 1,
+          },
+        },
+      },
+    },
+  },
+];
+
+export default function WaiterOrder() {
+  const [value, setValue] = React.useState('');
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+
+  const [expanded, setExpanded] = React.useState('');
+
+  const handleChangeExpanded = (panel) => (event, newExpanded) => {
+    setExpanded(newExpanded ? panel : false);
+  };
+
+  function getProductParams(product) {
+    let params = [];
+    for (let param in product.params) {
+      params.push(product.params[param]);
+    }
+
+    function getParamOptions(param) {
+      let options = [];
+      for (let option in param.options) {
+        options.push(param.options[option]);
+      }
+
+      return (
+        <div>
+          {options.map((option) => (
+            <FormControlLabel
+              key={option.label}
+              control={
+                <Checkbox checked={option.default} name={option.label} />
+              }
+              label={`${option.label} $${option.price}`}
+            ></FormControlLabel>
+          ))}
+        </div>
+      );
+    }
```

Comment:
To całe chyba moża wydzielić poza ciało funckji `getProductParams` Będzie czytelniej. Po drugie nazwa funckji nie pokrywa się z tym co robi. Obie funkcjie nie pobierają jak sugeruje nazwa, ale raczej renderują jakieś rzeczy

Reply:
Done